js/application.js

```diff
@@ -1,3 +1,3 @@
-App = Ember.Application.create();
+window.Abs = Ember.Application.create();
 
-App.ApplicationAdapter = DS.FixtureAdapter.extend();
\ No newline at end of file
+Abs.ApplicationAdapter = DS.FixtureAdapter.extend();
```

Comment:
it just for train, and I want to create a new namespace 


     On Monday, September 28, 2015 4:38 PM, Ruan Naude <notifications@github.com> wrote:
   

 In js/application.js:> \ No newline at end of file
> +Abs.ApplicationAdapter = DS.FixtureAdapter.extend();
Why did you rename the App to Abs?
And also why did you do window.Abs?—
Reply to this email directly or view it on GitHub.